packages/core/lib/blockUtils.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('lodash')
 const webpackMerge = require('webpack-merge')
+const webpackBlocksMerge = webpackMerge.smartStrategy({ 'resolve.extensions': 'prepend' })
```

Comment:
When putting it to `resolve`, we start introducing a lot of small different merge behaviors.
`webpack-blocks` allows you to define your own strategies and I think it makes sense to make use of it, as it alters only the object paths defined.

But yeah, we can still refactor this in case there are a lot of different merging strategies we want to have. 
You could also argue as well that this isn't core behavior and some people might want to write something that actually replaces / appends / do other crazy stuff.

Going to change ✅